### DISCUSSION TITLE
Zenon Network Community Video Contest

### DISCUSSION THREAD

**Discussion by sugoibtc** [coinselor/twitter#12] [Tweet Ideas]
🎥 Zenon Network Video Contest

We're calling all creators! Submit your short video explaining what makes @Zenon_Network stand out.

📹 Any format works - YouTube, MP4, GIF, etc.

⏳ Deadline: 11 Feb 2024 

🏆 Prize: 150 ZNN for the best video

Get started below 👇
--

Retweet https://twitter.com/Learn_Zenon/status/1748770462887256333

**Comment by coinselor** [comment]
Edited the format line, GIF makes no sense, it's an audioless format.


```
🎥 Zenon Network Video Contest

We're calling all creators! 

Submit your short video explaining what makes @Zenon_Network stand out.

📹 Any platform, format or style - YouTube, TikTok, etc

⏳ Deadline: 11 Feb 2024

🏆 Prize: 150 ZNN for the best video

Get started below 👇
---
https://twitter.com/Learn_Zenon/status/1748770462887256333
```

**Reply by 0x3639** [reply]
Should we link to twitter post or forum post?

**Reply by sugoibtc** [reply]
Linking to the forum might be better as it would be less friction. What do you guys think?

**Reply by 0x3639** [reply]
I'm cool either way.  Maybe someone other than @coinselor should submit the tweet?

**Comment by coinselor** [comment]
Successful tweet: https://x.com/Zenon_Network/status/1750508533932593582?s=20
